<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>denariiwalet</class>
 <widget class="QWidget" name="denariiwalet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Wallet Syncing: Syncing</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Denarii balance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dashboard_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Dashboard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stake_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Stake</string>
   </property>
  </widget>
  <widget class="QPushButton" name="borrow_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Denarii</string>
   </property>
  </widget>
  <widget class="QPushButton" name="request_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Request Denarii</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
